src/schneider_lab_to_nwb/zempolich_2024/zempolich_2024_intrinsic_signal_imaging_interface.py

```diff
@@ -74,20 +74,20 @@ def add_to_nwbfile(self, nwbfile: NWBFile, metadata: dict):
             name=isoi_metadata["Module"]["name"],
             description=isoi_metadata["Module"]["description"],
         )
-        raw_image = GrayscaleImage(
-            name=isoi_metadata["RawImage"]["name"],
-            data=raw_image_array,
-            description=isoi_metadata["RawImage"]["description"],
+        overlaid_image = RGBImage(
```

Comment:
Can you determine the nwb type (RGBImage vs GrayscaleImage) from the PIL object?

That would make the pipeline more robust in case they are doing the conversion themselves. 

In the same vein, if there are only two images I would suggest just using the file_path of the image instead of hardcoding the name.

Reply:
Both images should always be RGBImages, so I don't think I need to worry about that one.

But I can update to use 2 image_paths instead of a folder_path, I can see how that would be a bit easier to work with.